src/pages/[...params].js

```diff
@@ -14,8 +14,16 @@ const Fallback = () => {
       router.push('/login/');
     }
 
+    /* 
+    Remove the "params" query parameter added by Next 
+    (which holds the current path, based on `[...params]` in the filename)
+    because it causes infinite redirects as it keeps adding this query parameter to the iframe's URL 
+    */
+    const queryParameters = { ...query };
+    delete queryParameters.params;
```

Comment:
You could also just keep grab what you need and ignore params

```js
  const { query: {...queryWithoutParams, params} , asPath, ...router } = useRouter();
```

It's an example where using deeper restructuring would make sense, otherwise, you would need to create an unused intermediate variable